bundles/org.openhab.binding.sunsynk/src/main/java/org/openhab/binding/sunsynk/internal/classes/Inverter.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sunsynk.internal.classes;
```

Comment:
This looks like a regular dto that is used to communicatie with remote API, so better to move this to the apo/dto package.

Reply:
This is used to  hold the Config of the inverter on the openHAB side are you sure  you want me to move it. There will be nothing in classes?